src/main/java/com/celements/common/observation/listener/LocalEventListener.java

```diff
@@ -15,26 +22,81 @@
 @Component(LocalEventListener.NAME)
 public class LocalEventListener extends org.xwiki.observation.remote.internal.LocalEventListener {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(LocalEventListener.class);
+  private Logger LOGGER = LoggerFactory.getLogger(LocalEventListener.class);
 
   public static final String NAME = "observation.remote";
 
+  final Map<Class<? extends Event>, LogCounter> logCountMap = new ConcurrentHashMap<>();
+
   @Override
   public String getName() {
     return NAME;
   }
 
   @Override
   public void onEvent(Event event, Object source, Object data) {
-    if (event.getClass().isAnnotationPresent(Local.class)) {
+    Class<? extends Event> type = event.getClass();
+    if (type.isAnnotationPresent(Local.class)) {
       LOGGER.info("skipping local event '{}'", event.getClass());
     } else {
       try {
         super.onEvent(event, source, data);
       } catch (Exception exc) {
-        LOGGER.error("failed to notify RemoteObservationManager", exc);
+        if (containsNotSerializableException(exc)) {
```

Comment:
Note: `NotSerializableException` is a checked exception and not in the signature of onEvent, hence I cannot simply add a `catch(NotSerializableException nse)` block but have to analyse the given `Exception`